playback/src/decoder/passthrough_decoder.rs

```diff
@@ -29,19 +29,19 @@ where
         return Err(DecoderError::PassthroughDecoder("Invalid Data".into()));
     }
 
-    Ok(pck.data.into_boxed_slice())
+    Ok(pck.data.to_vec())
```

Comment:
```suggestion
    Ok(pck.data)
```

I think that `to_vec` might be superfluous.

Apart from that, this looks good to me, but my opinion doesn't really matter. ;-)